### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrey1104**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,73 @@ class Cargo:
...
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords
+        if self.coords is None:
+            self.coords = [0, 0]
```
Comment (viktoria-rybenchuk) [inline app/main.py:17]: use  ternary operator  for handling the initialization of self.coords, it  makes  the code shorter and more straightforward

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,73 @@ class Cargo:
...
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
```
Comment (viktoria-rybenchuk) [inline app/main.py:11]: specify type annotation for coords

File: app/main.py [hunk 3]
```diff
@@ -2,4 +2,73 @@ class Cargo:
...
+        self.coords[2] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            max_load_weight: int,
+            current_load: Cargo,
```
Comment (viktoria-rybenchuk) [inline app/main.py:59]:
```suggestion
            current_load: Cargo | None,
```

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**